cmd/crictl/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/urfave/cli"
+	"google.golang.org/grpc"
+)
+
+const (
+	DefaultTimeout = 10 * time.Second
```

Comment:
I think `DefaultTimeout` doesn't need to be exported, it also breaks validations because of no comments.